src/modules/editTodos.js

```diff
@@ -0,0 +1,26 @@
+import Todos from './Todos.js';
+
+const todos = new Todos();
+
+const editTodos = () => {
+  const todoText = document.querySelectorAll('.todoText');
+  todoText.forEach((todoTxt) => {
+    const eventClick = () => {
+      todoTxt.readOnly = false;
+      todoTxt.style.background = '#c2e9f0';
+    };
+    todoTxt.addEventListener('click', eventClick);
+    const eventFocus = () => {
+      todoTxt.style.background = 'none';
+      todos.allTodos.forEach((task) => {
+        if (task.index === parseInt(todoTxt.dataset.index, 10)) {
+          task.description = todoTxt.value;
+          todos.saveTasks();
+        }
+      });
+    };
+    todoTxt.addEventListener('focusout', eventFocus);
+  });
+};
```

Comment:
- [ ] I kindly suggest sticking to the requirement as much as possible. When you click on the detail icon, it should disappear, and the delete icon should appear. Also, note that the list item should have turned into a textible field. So far that is not functional 😊
- [ ] check screenshot below and compare it to the design template provided. 

![image](https://user-images.githubusercontent.com/92299875/221852991-728c01ec-9d44-4b90-8998-503a1baa21b0.png)